internal/page/app_data.go

```diff
@@ -30,9 +30,13 @@ type AppData struct {
 	LoginSessionEmail    string
 	Permission           actor.Permission
 	LoggedInSupporterID  string
-	IsDonorPage          bool
-	LpaType              actor.LpaType
-	DonorFullName        string
+	SupporterData        SupporterData
```

Comment:
Very minor thing, but could you make this `*SupporterData` so it is nil by default. Then it is very obvious when it isn't present

Reply:
and you'd be able to remove `IsDonorPage` here and use `if .SupporterData` in templates for the same effect